src/main/resources/ehcache.xml

```diff
@@ -0,0 +1,38 @@
+<config
+        xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+        xmlns='http://www.ehcache.org/v3'
+        xsi:schemaLocation="
+            http://www.ehcache.org/v3
+            http://www.ehcache.org/schema/ehcache-core-3.9.xsd">
+    <persistence directory="spring-boot-ehcache/cache" />
+
+
+
+
+    <cache-template name="default">
+        <expiry>
+            <ttl unit="seconds">30</ttl>
+        </expiry>
+        <listeners>
+            <listener>
+                <class>org.mifos.identityaccountmapper.util.CacheEventLogger</class>
+                <event-firing-mode>ASYNCHRONOUS</event-firing-mode>
+                <event-ordering-mode>UNORDERED</event-ordering-mode>
+                <events-to-fire-on>CREATED</events-to-fire-on>
+                <events-to-fire-on>EXPIRED</events-to-fire-on>
+                <events-to-fire-on>EVICTED</events-to-fire-on>
+            </listener>
+        </listeners>
+        <resources>
+            <heap>1000</heap>
```

Comment:
these values should be configurable from deployment

Reply:
ehcache.xml file is not supporting placeholders.
Can you suggest any alternate way to make this configurable?

Reply:
https://stackoverflow.com/questions/37301017/accessing-spring-environmental-variables-in-ehcache-xml